carma_wm_ctrl/src/WMBroadcasterNode.cpp

```diff
@@ -60,6 +60,7 @@ int WMBroadcasterNode::run()
   map_update_pub_ = cnh_.advertise<autoware_lanelet2_msgs::MapBin>("map_update", 200, [this](auto& pub){ wmb_.newUpdateSubscriber(pub); });
   //Route Message Publisher
   control_msg_pub_= cnh_.advertise<cav_msgs::TrafficControlRequest>("outgoing_geofence_request", 1, true);
+  TCR_timer = cnh_.createTimer(ros::Duration(10), &WMBroadcaster::TCR_timerCallBack,&wmb_ );
```

Comment:
This looks like the same timer as that done on line 86? Why would this code work and that would fail?

Reply:
The lambda callback seems executed for one time only on carma-platform start up and carma_wm_ctl run() is called.
This is the local testing debug info.
Time-1 is this PR change
Timer and 0 is the line 86 print log.
![image](https://user-images.githubusercontent.com/62157949/119913418-e1ef7e00-bf2b-11eb-8af1-469537b39b27.png)

Reply:
I see posts where lambda callback is also used for ros::timer, but I do not find a good explanation why this lambda function is not executed every 10s in this case.

Reply:
@dan-du-car Looking at the lambda callback it looks like the address for wmb_ is being passed in as the oneshot argument for the timer. This is likely casting to true for that boolean argument preventing that callback from triggering more than once. My guess is you could replace &wmb_ on line 89 with false and it would work.

Reply:
Thanks!